packages/node_modules/nav-frontend-lesmerpanel/src/lesmerpanel.tsx

```diff
@@ -102,9 +102,11 @@ class Lesmerpanel extends React.Component<LesMerPanelBaseProps, LesMerPanelBaseS
                 className={lesMerPanelCls(this.props)}
                 {...domProps}
             >
-                <div className="lesMerPanel__intro">
-                    {intro}
-                </div>
+                {
+                    !!intro && <div className="lesMerPanel__intro">
```

Comment:
Trenger du `!!` her? 🤔

Reply:
Kanskje ikke når du sier det. Pleier å gjøre det for å force boolean utrykk ut i javascripten. Det vil vel funke fint her uten. Hva er preferansen?

Reply:
Er vel ikke noe problem akkurat, men tenker det kan være greit å holde ting så enkelt som mulig - og da heller droppe ting som egentlig ikke er nødvendig 🙂

Reply:
Fixed